main.js

```diff
@@ -1 +1,96 @@
-var x;
+var Product = function () {
+    this.w = 0;
+    this.h = 0;
+};
+Product.prototype.render = function () {
+    var node = document.createElement('div');
+    node.setAttribute('style', 'border: 1px solid #000000; padding: 2px; margin: 2px; width:'
```

Comment:
Is it necessary to do inline styling for static attributes? Would suggest adding a class to the div and do styling in a css file